sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -209,18 +277,46 @@ PollerFlux<AnalyzeActionsOperationDetail, AnalyzeActionsResultPagedIterable> beg
                     .setTasks(getJobManifestTasks(actions));
             analyzeBatchInput.setDisplayName(actions.getDisplayName());
             final boolean finalIncludeStatistics = options.isIncludeStatistics();
+
+            if (service != null) {
+                return new PollerFlux<>(
+                    DEFAULT_POLL_INTERVAL,
+                    activationOperation(
+                        service.submitJobWithResponseAsync(
+                            new AnalyzeTextJobsInput()
+                                .setDisplayName(actions.getDisplayName())
```

Comment:
Can actions be null here?

Reply:
Actions can't be null. Adding a NPE check now